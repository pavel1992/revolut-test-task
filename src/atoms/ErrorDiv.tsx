import styled from 'styled-components';

import { colors } from '../styleConstants/colors';

export const ErrorDiv = styled.div`
  margin: auto 10px;
  color: ${colors.errorRed};
`;
