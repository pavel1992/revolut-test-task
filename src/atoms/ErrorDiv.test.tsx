import * as React from 'react';
import renderer from 'react-test-renderer';

import { ErrorDiv } from './ErrorDiv';


test('renders ErrorDiv correctly', () => {
  const render = renderer
    .create(<ErrorDiv>111</ErrorDiv>)
    .toJSON();
  expect(render).toMatchSnapshot();
});
