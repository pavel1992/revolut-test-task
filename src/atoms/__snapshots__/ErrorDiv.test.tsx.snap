// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders ErrorDiv correctly 1`] = `
.c0 {
  margin: auto 10px;
  color: #FF1F1F;
}

<div
  className="c0"
>
  111
</div>
`;
